### PR TITLE
Add "from" to Mailgun action

### PR DESCRIPTION
**Pull request by omeryarn**

### CODE REVIEW

**Comment by omeryarn** [comment]
Fixed the default value

File: docs/Actions.md [hunk 1]
```diff
@@ -1470,6 +1470,7 @@ mailgun(
   postmaster: "MY_POSTMASTER",
   apikey: "MY_API_KEY",
   to: "DESTINATION_EMAIL",
+  from: "My awesome app",
```
Comment (asfalcone) [inline docs/Actions.md]: Can you make the example here a little bit more generic? like EMAIL_FROM_NAME or something? "My awesome app" doesn't provide enough context for the `from` value for users reading this documentation.

**Comment by asfalcone** [comment]
Thanks for adding this enhancement! 📬

I added 1 more comment. Please squash all commits on this branch and it will be ready to merge. 🎉

**Comment by omeryarn** [comment]
Added a new commit for the actions doc

**Comment by asfalcone** [comment]
Thanks! Please squash the commits down to 1 and it will be ready to go!

**Comment by omeryarn** [comment]
Can't, they're not consecutive. What's the issue?

**Comment by asfalcone** [comment]
@omeryarn We prefer that PRs have a single commit when they are merged.

**Comment by 4brunu** [comment]
If you want, I can create a pull request with only one commit, because this enhancement would be useful to me.

**Comment by omeryarn** [comment]
Sure, go ahead

On Wed, Jan 13, 2016 at 12:10 PM, 4brunu <notifications@github.com> wrote:

> If you want, I can create a pull request with only one commit, because
> this enhancement would be useful to me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fastlane/fastlane/pull/1061#issuecomment-171240963>.
>



-- 
עומר

**Comment by KrauseFx** [comment]
Closed in favor of https://github.com/fastlane/fastlane/pull/1091

**Pull request closed without merging**